inference.py

```diff
@@ -312,7 +318,7 @@ def inference_multimer_model(args):
 
     # Save the relaxed PDB.
     relaxed_output_path = os.path.join(args.output_dir,
-                                        f'{tag}_{args.model_name}_relaxed.pdb')
+        f'{output_prefix}_{args.model_name}__relaxed.pdb')
```

Comment:
Why there are 2 '_' before 'relaxed'?

Reply:
Thanks for spotting the typo, should be fixed now.